pyfr/solvers/base/elements.py

```diff
@@ -212,6 +212,10 @@ def set_backend(self, backend, nscalupts, nonce, linoff):
         self.scal_upts_inb = inb = backend.matrix_bank(self._scal_upts)
         self.scal_upts_outb = backend.matrix_bank(self._scal_upts)
 
+        # Allocate and bank the storage required to store the solution grads
+        # which will be used by post-processing plugins
+        self.vect_upts_inb = backend.matrix_bank([self._vect_upts])
```

Comment:
No need to bank this.

Reply:
Do you then suggest to use `self.vect_upts_ib = self._vect_upts`? I'm sorry if these questions are a bit silly, I'm not fully used to the way in which data is accessed in PyFR

Reply:
Yes; this is on account of the fact that there is only one self._vect_upts array (unlike scalar arrays where due to the RK schemes there can be a bunch).